Stresser/resource.h

```diff
@@ -0,0 +1,16 @@
+//{{NO_DEPENDENCIES}}
+// Microsoft Visual C++ generated include file.
+// Used by Stresser.rc
+//
+#define FAKE_PROCESS                    4
```

Comment:
CHORE for both of us:
Move this constant (FAKE_PROCESS) to StresserApplication object.